PhoneAuth/FirebasePhoneAuthUI/FUICodeField.m

```diff
@@ -179,6 +186,10 @@ - (UITextContentType _Null_unspecified)textContentType {
   return nil;
 }
 
+- (void)setTextContentType:(_Null_unspecified UITextContentType)textContentType {
```

Comment:
Could you explain more how this fix the issue? why using _Null_unspecified

Reply:
This doesn't fix the issue, it just conforms better to the protocol since the `textContentType` property is not readonly.

Reply:
Does it fix #775?

Reply:
The fix is in lines 62-68.

Reply:
Yeah, sounds good. I understand it now.